src/core/src/sys/connect.rs

```diff
@@ -0,0 +1,163 @@
+// Lumol, an extensible molecular simulation engine
+// Copyright (C) 2015-2016 G. Fraux — BSD license
+
+//! Data about bonds and angles in the system.
+use std::cmp::{min, max};
+
+/// A `Bond` between the atoms at indexes `i` and `j`
```

Comment:
I'd go with `particles` instead of `atoms` in all descriptions. Makes it easier to understand since we also have the `Particle` struct.